includes/class-ur-ajax.php

```diff
@@ -1404,6 +1405,24 @@ public static function create_form() {
 			)
 		);
 	}
+	/**
+	 * Email setting status
+	 */
+	public function email_setting_status() {
```

Comment:
Make the function static. It will throw error otherwise.

Reply:
He has fixed it